### DISCUSSION TITLE
thoughts on removing hacks/ entirely?

### DISCUSSION THREAD

**Discussion by PatheticMustan** [Prodigy-Hacking/ProdigyMathGameHacking#1167] [(D) Developer]
at this point, hacks/ is just super outdated. What are your thoughts on removing it entirely, @ArcerionDev, @YamaHacks, @BoredFishRE, @MelnCat?

Also yama wtf is hacks/instantDeath/, why does it still exist

**Comment by hostedposted** [comment]
I think we should

**Comment by afkvido** [comment]
NO PLEASE DONT DO NOT REMOVE THE HACKS PLEASE MUSTAN FOR THE LOVE OF ME AND THE 80K+ USERS <3

**Reply by hostedposted** [reply]
It won't remove the hacks. It will only remove the folder named hacks as it useless.

**Comment by TheFinality** [comment]
I think we should. First of all, there are too many skids, which means we get a shit ton of complaints. Next, the devs have a life, so let them relax smh. Last, as said above, we can let the devs relax, or they will get mental health problems. 

there i wrote a mini essay

**Reply by ArcerionDev** [reply]
this isnt getting rid of hacks completely

**Comment by PatheticMustan** [comment]
to clarify, this wouldn't actually remove hacks, it would just remove the outdated folder "hacks".
Reasoning being that we don't actually use the folder, and most of the information it contains is outdated/incorrect. unless another maintainer is interested in maintaining that folder, we might as well remove it

**Reply by TheFinality** [reply]
oh

**Comment by ArcerionDev** [comment]
I'm in favor of this.

**Reply by ghost** [reply]
im in favor of rick astley

**Comment by ghost** [comment]
i have a better idea so shotgun to face and hack bad

**Comment by ThatLegend** [comment]
NO REMOVING HACKS. I need prodigy hacks to live.

**Reply by ghost** [reply]
bruh you havent read anything this person said are you fucking existing and a normal human being because if you are everyone should shame you for not being perfect /s

**Reply by hostedposted** [reply]
> bruh you havent read anything this person said are you fucking existing and a normal human being because if you are everyone should shame you for not being perfect /s

Can you add some commas?

**Reply by PatheticMustan** [reply]
Bruh! You have not yet read anything this person has said. Are you fucking existing and a normal human being? If you are, everyone should shame you for not being perfect. /s

**Reply by hostedposted** [reply]
Thank you

**Comment by ThatLegend** [comment]
wdym

**Reply by ghost** [reply]
sex
